src/Cake.Tfs/TfsAliases.PullRequest.cs

```diff
@@ -55,6 +55,34 @@ public static partial class TfsAliases
             return null;
         }
 
+        /// <summary>
+        /// Gets a Team Foundation Server or Azure DevOps pull request using the settings obtained from predefined build variables.
```

Comment:
We should mention in the summary that the alias is only intendet to be used while running in an Azure Pipelines build.